src/invocation/InvocationService.ts

```diff
@@ -222,24 +269,45 @@ export class InvocationService {
 
         if (clientMessage.hasFlags(BitsUtil.LISTENER_FLAG)) {
             setImmediate(() => {
-                if (this.eventHandlers[correlationId] !== undefined) {
-                    this.eventHandlers[correlationId].handler(clientMessage);
+                if (this.eventHandlers.has(correlationId)) {
```

Comment:
Minor: We can try to get the handler first, and if it is not undefined, call it

Reply:
Addressed in f255caaf6d8fc835c8f04eab96d7b8b59b58e4c3